src/test/models/inventory.ts

```diff
@@ -0,0 +1,36 @@
+// Tests for discriminators and refPaths
+import { prop, Ref, Typegoose } from '../../typegoose';
+
+export  class Scooter extends Typegoose {
+    @prop()
+    makeAndModel: string
```

Comment:
could you add `;` to the line endings (all in the file) to fit to fit the others?